newsControllers/errorHandlingControllers.js

```diff
@@ -0,0 +1,20 @@
+exports.handlePSQL400s = (error, request, response, next) => {
```

Comment:
Was not sure yesterday how to do proper error handling for this ticket, so decided to make this rather than nothing, though it did look unnecessary. Everything makes more sense with today's lecture.